modules/tinymce/src/themes/silver/main/ts/ui/menus/contextmenu/SilverContextMenu.ts

```diff
@@ -122,19 +123,16 @@ const getSelectedElement = (editor: Editor, e: PointerEvent) =>
 const shouldUseNodeAnchor = (editor: Editor, e: PointerEvent) => {
   const selector = Settings.getAvoidOverlapSelector(editor);
   if (isTriggeredByKeyboard(editor, e)) {
-    return true;
+    return false;
```

Comment:
This can't be done and doesn't do what we had said. With this change it'll use a point anchor instead which is incorrect. So as we've discussed previously, it should be using a selection anchor instead for keyboard activation. It should **always** be using a node anchor for the case below. Lastly it should use the point anchor for clicks/touches.

Reply:
I also still stand by what I had told you originally, that there's possibly a bug in the `isTriggeredByKeyboard` logic if it's returning true for Android devices.

Reply:
I've made the change passing the anchor type rather than `useAnchorNode: boolean`. Haven't figured out how to fix Android returning true for `isTriggeredByKeyboard`. Did some tests and it seems when you `longpress` it triggers `longpress` twice and it goes to `getPointAnchorSpec`. Then fires `contextmenu` event twice with `e.button === -1` and returns a SelectionAnchor. See below.

<img width="549" alt="longpress-android" src="https://user-images.githubusercontent.com/691570/128299803-d43bb81b-0f37-428e-9c55-5296d758114d.png">